servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/AlpnServerContext.java

```diff
@@ -53,7 +53,6 @@ private AlpnServerContext() {
                                       @Nullable final ConnectionAcceptor connectionAcceptor,
                                       final StreamingHttpService service,
                                       final boolean drainRequestPayloadBody) {
-        assert config.h1Config() != null && config.h2Config() != null;
```

Comment:
I will restore this assert. The current code allows overriding the ALPN protocols via SslConfig. The idea was folks could re-order their ALPN preference for each individual SNI map entry on the server, but todo this safely we would need to coordinate the http configuration with the SNI selection as well. since we don't need this now I'll punt.